addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/SerialMessage.java

```diff
@@ -154,7 +158,13 @@ public SerialMessage(int nodeId, byte[] buffer) {
         }
         this.priority = SerialMessagePriority.High;
         this.messageType = buffer[2] == 0x00 ? SerialMessageType.Request : SerialMessageType.Response;
-        this.messageClass = SerialMessageClass.getMessageClass(buffer[3] & 0xFF);
+        final int messageClassId = buffer[3] & 0xFF;
+        this.messageClass = SerialMessageClass.getMessageClass(messageClassId);
```

Comment:
My intention was to differ between never seen (as developer point of view) serial message keys and message keys we have already seen but are not handled.
We already know ("now") that this USB device generates messages with the key 0x0A.
Logging an exception for something we can ignore silently (can we?) can confuse the user.